docs/release/releases/release-1.6.md

```diff
@@ -4,33 +4,33 @@
 
 The following table shows the preliminary dates for the `v1.6` release cycle.
 
-| **What**                                             | **Who**      | **When**                   | **Week** |
-|------------------------------------------------------|--------------|----------------------------|----------|
-| Start of Release Cycle                               | Release Lead | Monday 31st July 2023      | week 1   |
+| **What**                                             | **Who**      | **When**                    | **Week** |
```

Comment:
Thx for this fix. I didn't want to annoy anyone with nitpicking when the KubeCon line was added :)